src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/CorsTests.cs

```diff
@@ -9,15 +10,75 @@
 
 namespace Microsoft.AspNetCore.Mvc.FunctionalTests
 {
-    public class CorsTests : CorsTestsBase<CorsWebSite.StartupWith21Compat>
+    public class CorsTests : IClassFixture<MvcTestFixture<CorsWebSite.Startup>>
```

Comment:
What I did here is more like renaming `CorsTestBase` to `CorsTest` and using the `CorsEndpointRoutingTests` choice of `Startup` class.